text/0000-project-groups.md

```diff
@@ -0,0 +1,385 @@
+# Project Group RFC
+
+# Summary
+
+- Formalize project groups as groups dedicated to completing specific projects
+  within the context of a Rust team
+- Project groups are created via an RFC and have a "parent team" (or
+  multiple teams)
+- The groups then drive the project to completion, e.g. by authoring follow-up
+  RFCs and doing design work.
+- Each project group typically has:
+  - A charter outlining the group's scope and goals.
+  - Appointed shepherds and team liasons.
+  - An associated repository.
+  - Regular meetings.
+  - Dedicated streams on Discord/Zulip/etc.
+- Define working groups to refer to the "domain working groups" that are created
+  to explore particular domains, such as embedded, CLI, etc.
+  - They have a charter and defined leads but operate more independently from
+    the Rust teams.
+- Define community group as the term for groups not formally affiliated with the
+  Rust project.
+
+# Motivation
+
+Working groups were not created through the RFC process, as such there's not
+much documentation on exactly what a working group does, what are its
+responsibilities, and also importantly setting expectations or goals. There is
+one definition of "Working Group" available in [rust-lang#54445]; quoted below.
+
+> A Rust Working Group is a set of people working at common purpose. Working
+> Groups are associated with a Rust team. Unlike a Rust Team, Working Groups
+> don't have "formal decision making power", though often they are charged with
+> drawing up recommendations, RFCs, or other documents for the teams (which is
+> then intended to make the final decision).
+
+While this definition is true in the broad sense, it does not define the
+difference between what has come to be called "Domain Working Groups" and
+"Team Working Groups". Additionally the community has also adopted the Working
+Group terminology for organising and working on areas they are interested in,
+independent of The Rust Programming Language's Working Groups. It's great that
+we have a community able to self organise in this way, however it has led to
+some confusion over who is supporting these efforts, and whether they're
+considered _"official"_ Working Groups.
```

Comment:
Is this actually a problem? I think it ultimately it boils down to people wanting to know if their efforts will make a difference, or if a group has the power to make decisions. Every group can make decisions in it's own problem domain, but when it comes to overall project choices there's a lot of social dynamics involved that go beyond the question if a working group is official or not.  I think focussing on that would be more important

Reply:
The consideration of whether a working group is official is more about access and organisational policy rather than about people's choice's be considered authoritative. There's a lot of places where we have to draw a line somewhere. One example of this is all hands attendance. Official working group leads get invited, unofficial ones do not.